### ISSUE TITLE
How woudl you handle the case where your actors needs constructor injection

### ISSUE THREAD

**Issue by paulvanbladel** [ioanb7/goldenshoe#2]
Very nice sample.
I hope you don't mind if I ask you a question.
How would you handle the DI scenario where your actors themselves would need constructor injection of other services?

Cheers
paul.

**Comment by ioandev** [comment]
Hi Paul,

Do you mean Dependency Injection inside the asp.NET core app ?

You need to have a reference to the Dependency Injection the app is using. Haven't tested, but this response looks promising: https://stackoverflow.com/a/38235154

Ioan.

**Comment by paulvanbladel** [comment]
Yes for getting actors who in turn have dependencies on constructor, from the standard .net core DI container inside asp.net core app. I learned in the mean time from github issue on akka.net repo that there is no support for standard asp.net DI core (there is support though for autofac and others)
The point is that actors heavily depend on magical PROPS, a concept which is quite incompatible with DI.
Especially when you start using supervisor actors which create other actors.
I' think i'll start looking more into the direction of microsoft orleans where DI just works :)
Thanks for you help !
paul.
